src/mcts/node.cc

```diff
@@ -181,8 +181,17 @@ bool Node::TryStartScoreUpdate() {
 void Node::CancelScoreUpdate() { --n_in_flight_; }
 
 void Node::FinalizeScoreUpdate(float v) {
-  // Recompute Q.
+  // Recompute MCTS Q.
   q_ += (v - q_) / (n_ + 1);
+  // Recompute MinMax Q.
+  if (n_ == 0) {
+    minmax_q_ = v;
+  } else {
+    for (Node* child : ChildNodes()) {
```

Comment:
I noticed that I'm missing the `minmax_q_ = 1.0f` initialization here, but also it would be better to use `min_element` from C++ std.